pygame_menu/examples/window_resize.py

```diff
@@ -7,55 +7,95 @@
 """
 
 import pygame
+import random, sys, math, time
 import pygame_menu
+from pygame_menu import themes
```

Comment:
Hi. Thanks for introducing a new example.

Please create a new file rather than updating the existing one with better documentation to show what is the intention of this script. Also, make sure you add that also to test/test_examples.py. Looking forward new updates,

Pablo